lib/util/cache/repository/index.ts

```diff
@@ -0,0 +1,58 @@
+import * as fs from 'fs-extra';
+import { join } from 'upath';
+import { RenovateConfig, RepositoryCacheConfig } from '../../../config/common';
+import { logger } from '../../../logger';
+import { PackageFile } from '../../../manager/common';
+
+export interface BaseBranchCache {
+  sha: string; // branch commit sha
+  configHash: string; // object hash of config
+  packageFiles: PackageFile[]; // extract result
+}
+
+export interface Cache {
+  init?: {
+    configFile: string;
+    contents: RenovateConfig;
+  };
+  scan?: Record<string, BaseBranchCache>;
+}
+
+let repositoryCache: RepositoryCacheConfig = 'disabled';
+let cacheFileName: string;
+let cache: Cache = {};
```

Comment:
```suggestion
let cache: Cache = Object.create({});
```
Do not inherit from `Object`